File: ClassForgeAPI/src/main/java/com/classforge/api/ClassForgeAPI/dao/Info.java
package com.classforge.api.ClassForgeAPI.dao;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
@Table(name = "info")
public class Info {
    @Id
    @GeneratedValue(strategy = jakarta.persistence.GenerationType.IDENTITY)
    private Integer id;

    @Column(name="name")
    private String name;

    @Column(name="surname")
    private String surname;

    @Column(name="photo")
    private String photo;

    @Column(name="description")
    private String description;

    @OneToOne
    @JoinColumn(name="user_id")
    private User user;
}

File: ClassForgeWeb/ClassForgeWeb/src/app/shared/info.ts
export class Info{
  public name: string="";
  public surname: string="";
  public photo: string="";
  public description: string="";
  public user_id: number=0;
  public year: number;
  constructor(data: any) {
    this.name = data.name;
    this.surname = data.surname;
    this.photo = data.photo;
    this.description = data.description;
    this.user_id = data.user_id;
    this.year = data.year;
  }
}
